src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -32,6 +32,15 @@ GcsResourceManager::GcsResourceManager(
       local_node_id_(std::move(local_node_id)),
       cluster_task_manager_(std::move(cluster_task_manager)) {}
 
+absl::Time GcsResourceManager::GetLastResourceUsageUpdateTime(
```

Comment:
Since any successful grpc call to raylet or grpc call from raylet can count towards a successful health check. We should consider track it in lower level common raylet client and gcs server: candidate is node_manager_client.h